runatlantis.io/docs/deployment.md

```diff
@@ -481,6 +514,27 @@ containers:
           key: webhook-secret
 ```
 
+#### Gitea
+
+```yaml
+...
```

Comment:
This will break the yaml color coding if not remove

```suggestion
```

Reply:
Fine by me, but then all the other existing code snippets are also breaking the color coding. :smile:

Reply:
Oof. Thanks for fixing. At least now we will have a good example to point to help improve docs further. 😁